docs/admin/bootstrap-checks.txt

```diff
@@ -24,6 +16,18 @@ CrateDB passes the bootstrap checks and can perform optimally.
 If you are binding to the loopback address, the bootstrap checks will not be
 run, but it is still a good idea to follow these instructions.
 
+.. TIP::
+
+    If you are using Docker, these checks are dependent on the host system. You
+    must configure the host system appropriately if you wish to run CrateDB on
+    Docker. Consult the additional documentation on Docker :ref:`resource
+    constraints <resource_constraints>` for more information.
```

Comment:
should there be a url here?

Reply:
the :ref:`resource constraints <resource_constraints>` bit is turned into a url. it's using `resource_constraints` as a fragment identifier (i.e. a section elsewhere in the docs has this assigned as an ID)